### DISCUSSION TITLE
Observability?

### DISCUSSION THREAD

**Discussion by eescapa-carbonre** [mkdocs/mkdocs#3243] [Q&A] [answered]
Hello,

Has anyone tried implementing observability/monitoring tools (such as [Honeycomb](https://www.honeycomb.io/)) to track how users are using your documentation? I'm very new to implementing Markdown and I haven't seen any integrations or plugins related to telemetry, wanted to see if anyone here has already attempted it. Thank you!

**Comment by pawamoy** [comment]
Hmm I'm not sure telemetry makes sense for static websites? Maybe you're thinking about analytics instead? The Material for MkDocs theme [supports Google Analytics](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/?h=analytics) out of the box, but you can use alternatives like [Matomo](https://matomo.org/) as well.

**Reply by eescapa-carbonre** [reply]
Ohh I see, is MkDocs entirely static even when you click around to navigate? I don't care too much about how long pages or assets take to load, more which pages users are opening and what they're searching for (which I see is supported for Google Analytics, thank you for sending that link!) 

I'll play around with the [Custom Site Analytics](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/?h=analytics#custom-site-analytics) and see how far I get 👍

**Reply by pawamoy** [reply, marked as answer]
Yes, MkDocs generates static websites, there's no webserver in use when you deploy your site somewhere. Well, at least no webserver running application code: all the pages are already built and the HTML is directly sent to clients without per-user processing or else. It's like serving contents from a folder :)

So yeah analytics seem to be what you're looking for :+1:

**Reply by eescapa-carbonre** [reply]
Thank you!!
